### PR TITLE
docs: fix typos in README file

### PR DESCRIPTION
**Pull request by pbmchc**
Fixes typos in README file.
```diff
- ouside
+ outside
```
```diff
- vertacally
+ vertically 
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ Transfer to Clothing    -50.00
...
 
-The chart should show the percentage spent in each category passed in to the function. The percentage spent should be calculated only with withdrawals and not with deposits. Down the left side of the chart should be labels 0 - 100. The "bars" in the bar chart should be made out of the "o" character. The height of each bar should be rounded down to the nearest 10. The horizontal line below the bars should go two spaces past the final bar. Each category name should be vertacally below the bar. There should be a title at the top that says "Percentage spent by category".
+The chart should show the percentage spent in each category passed in to the function. The percentage spent should be calculated only with withdrawals and not with deposits. Down the left side of the chart should be labels 0 - 100. The "bars" in the bar chart should be made out of the "o" character. The height of each bar should be rounded down to the nearest 10. The horizontal line below the bars should go two spaces past the final bar. Each category name should be vertically below the bar. There should be a title at the top that says "Percentage spent by category".
```
Comment (naomi-lgbt) [inline README.md]:
```suggestion
The chart should show the percentage spent in each category passed in to the function. The percentage spent should be calculated only with withdrawals and not with deposits. Down the left side of the chart should be labels 0 - 100. The "bars" in the bar chart should be made out of the "o" character. The height of each bar should be rounded down to the nearest 10. The horizontal line below the bars should go two spaces past the final bar. Each category name should be written vertically below the bar. There should be a title at the top that says "Percentage spent by category".
```
A slight verbiage tweak.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me. Thank you for taking care of that.

File: README.md [hunk 2]
```diff
@@ -23,9 +23,9 @@ Transfer to Clothing    -50.00
...
 
-The chart should show the percentage spent in each category passed in to the function. The percentage spent should be calculated only with withdrawals and not with deposits. Down the left side of the chart should be labels 0 - 100. The "bars" in the bar chart should be made out of the "o" character. The height of each bar should be rounded down to the nearest 10. The horizontal line below the bars should go two spaces past the final bar. Each category name should be vertacally below the bar. There should be a title at the top that says "Percentage spent by category".
+The chart should show the percentage spent in each category passed in to the function. The percentage spent should be calculated only with withdrawals and not with deposits. Down the left side of the chart should be labels 0 - 100. The "bars" in the bar chart should be made out of the "o" character. The height of each bar should be rounded down to the nearest 10. The horizontal line below the bars should go two spaces past the final bar. Each category name should be written vertically below the bar. There should be a title at the top that says "Percentage spent by category".
```
Comment (RandellDawson) [inline README.md:28]: Does the word `vertically` even need to be here?  Below would imply vertically in my opinion.
```suggestion
The chart should show the percentage spent in each category passed in to the function. The percentage spent should be calculated only with withdrawals and not with deposits. Down the left side of the chart should be labels 0 - 100. The "bars" in the bar chart should be made out of the "o" character. The height of each bar should be rounded down to the nearest 10. The horizontal line below the bars should go two spaces past the final bar. Each category name should be below the bar. There should be a title at the top that says "Percentage spent by category".
```

Comment (pbmchc) [reply README.md:28]: From the user's perspective I would keep the longer version but given the fact that below the paragraph there is also an example visualization I think that both approaches are fully acceptable. So the decision is up to you 🙂

**Review by moT01 — APPROVED** [review]
LGTM @pbmchc 🎉 Congratulations on your first contribution to this freeCodeCamp repository 🎉 

I think I am just going to merge this as-is since it is how the writer originally intended it. Like mentioned, it works both ways.

**Pull request merged**